src/ResourceManager/Automation/Commands.Automation.Test/UnitTests/GetAzureAutomationDscNodeConfigurationTest.cs

```diff
@@ -0,0 +1,123 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Azure.Commands.Automation.Cmdlet;
+using Microsoft.Azure.Commands.Automation.Common;
+using Microsoft.Azure.Commands.Automation.Model;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.WindowsAzure.Commands.Common.Test.Mocks;
+using Microsoft.WindowsAzure.Commands.ScenarioTest;
+using Microsoft.WindowsAzure.Commands.Test.Utilities.Common;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using Moq;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Microsoft.Azure.Commands.ResourceManager.Automation.Test.UnitTests
+{
+    [TestClass]
```

Comment:
Marc, regarding the other comment below, 
we are in timecrunch. I haven't written scenario tests in powershell. if you think, we can do this in couple of hours, Please guide me with an example. Otherwise, I can rise an issue here and will fix in next iteration.

We have GQL tests and tested this change thoroughly. Therefore, we are confident about this change.